app/models/patient.rb

```diff
@@ -50,8 +50,12 @@ class Patient
   field :identifier, type: String
 
   # Contact-related info
-  enum :voicemail_preference, [:not_specified, :no, :yes]
-  enum :line, LINES # See config/initializers/env_vars.rb
+  field :voicemail_preference
+  enumerize :voicemail_preference, in: [:not_specified, :no, :yes], default: :not_specified
```

Comment:
I **think** we'll want to declare this an int by doing something like `in: [not_specified: 0, no: 1, yes: 2]` so we can avoid migrating the current data. I think.